src/new-components/text/text.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs, select } from '@storybook/addon-knobs'
+
+import { Stack } from '../stack'
+import { Text } from './text'
+
+const TextChapter = {
+    subtitle: 'Text',
+    sections: [{ sectionFn: TextStory, options: { showPropTables: false } }],
+}
+
+function TextStory() {
+    return (
+        <section className="story">
+            <Stack space="medium">
+                <Text size="xlarge" weight="strong" tone="secondary">
+                    Text, xlarge, strong, secondary (16px)
```

Comment:
Should `xlarge` use a larger font-size?

Reply:
Yep, both are 16px. I've created a post to align with design and you on this so we may make more changes here after 👍